core/block_validator_test.go

```diff
@@ -35,7 +35,7 @@ func testChainConfig() *ChainConfig {
 		Forks: []*Fork{
 			&Fork{
 				Name:  "Homestead",
-				Block: big.NewInt(1150000),
+				Block: big.NewInt(0),
```

Comment:
This is another thing, this is because tests are using `IsHomestead` and operates against blockchain of few blocks. Then expect it to be homestead chain in tests, so homestead is set to 0 nearly everywhere in tests (except situations where it expects different type of chain, this is a default config)